code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -711,8 +711,8 @@
 			LM.playsound_local(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
 
 /obj/item/integrated_circuit/input/ntnet_packet
-	name = "NTNet networking circuit"
-	desc = "Enables the sending and receiving of messages over NTNet via packet data protocol."
+	name = "wifi networking circuit"
```

Comment:
all these mentions of wifi, should it just be 'network' to be vague? or 'radio'? :eyes:

Reply:
The issue with 'radio' is that radios don't have device addresses. I could always just change it to 'wireless networking circuit'?